architecture/additional_concepts/authorization.adoc

```diff
@@ -880,6 +880,43 @@ in their SCC set.  This allows cluster administrators to run pods as any
 user by without specifying a `RunAsUser` on the pod's `SecurityContext`.  The
 administrator may still specify a `RunAsUser` if they wish.
 
+[[rbac-to-scc]]
+==== Role-Based Access to SCCs ====
+Starting with {product-title} 3.11, you can specify Security Context Constrains as a
+resource that is handled by RBAC. This allows you to
+scope the access to your SCCs to a certain project or cluster wide. Assigning users/groups
+directly to an SCC retains cluster-wide scope.
+
+To include access to SCCs for your role, you need to specify the following rule:
+.Role-Based Access to SCCs
+[source,yaml]
+----
+rules:
+  apiGroups:
+  - security.openshift.io <1>
+  resources:
+  - securitycontextconstraints <2>
+  verbs:
+  - use
+  resourceNames:
+  - myPermittingSCC <3>
+----
+<1> The API group which includes the `securitycontextconstraints` resource
```

Comment:
Do we point out apiGroup here because it's interesting or necessary to change (if so, when)?

Reply:
I was under the impression (after reading https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-groups ) that it's necessary to mention it to get access to the `apiGroup` elements, in this case the `securitycontextconstraints` resource.

If it's not important here, or if it's implied, we can remove it from the docs.